### PR TITLE
Session management

### PR DESCRIPTION
**Pull request by TimMikeladze**
This PR introduces code to create, update, and invalidate sessions.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -88,6 +89,82 @@ class RedisDBInterface {
...
+        userAgent,
+        ip,
+        updatedAt: Date.now(),
+      });
+      await pipeline.exec();
+    }
+  }
+  async invalidateSession(sessionId: string): Promise<void> {
+    if (this.redis.exists(`session:${sessionId}`)) {
+      this.redis.hmset(`session:${sessionId}`, {
```
Comment (grantga) [inline src/index.js]: You could also delete the key here?

Comment (TimMikeladze) [inline src/index.js]: Yeah I could. I need to give session expiration a bit more thought. Sessions provide a paper trail of user logins which can be useful for certain scenarios.

Comment (grantga) [inline src/index.js]: Yea that sounds good. We just don't want the key space to grow infinitely so will need to manage them in some way.

Comment (TimMikeladze) [inline src/index.js]: See #5

**Pull request merged**